src/App.jsx

```diff
@@ -1,11 +1,67 @@
 import './App.css'
+import axios from "axios";
+import {useEffect, useState} from "react";
+import PokemonInfoCard from "./components/PokemonInfoCard.jsx";
 
 function App() {
+    let allPokemonData = [];
+    const [endNumber, setEndNumber] = useState(20);
+    const [startNumber, setStartNumber] = useState(1);
+    const [fetchingCompleted, setfetchingCompleted] = useState(false);
+    const [pokemonData, setpokemonData] = useState([]);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            setfetchingCompleted(false);
+
+            try {
+                for (let i = startNumber; i < endNumber; i++) {
+                    const pokemonData = await axios.get("https://pokeapi.co/api/v2/pokemon/" + i + "/");
+                    allPokemonData.push(pokemonData.data);
+                }
+                setpokemonData(allPokemonData);
```

Comment:
```suggestion
                for (let i = startNumber; i < endNumber; i++) {
                    const pokemonData = await axios.get("https://pokeapi.co/api/v2/pokemon/" + i + "/");
                    setpokemonData(currentData => [...currentData, ...pokemonData.data]);
                }
```
Dan hoef je de andere array niet te gebruiken 😉